utils/ingress/ingress.go

```diff
@@ -11,18 +12,24 @@ import (
 	"k8s.io/client-go/discovery"
 
 	"github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1"
+	"github.com/argoproj/argo-rollouts/utils/defaults"
 	"github.com/argoproj/argo-rollouts/utils/diff"
+	"github.com/argoproj/argo-rollouts/utils/json"
 )
 
 const (
 	// CanaryIngressSuffix is the name suffix all canary ingresses created by the rollouts controller will have
 	CanaryIngressSuffix = "-canary"
 	// ManagedActionsAnnotation holds list of ALB actions that are managed by rollouts
 	ManagedActionsAnnotation = "rollouts.argoproj.io/managed-alb-actions"
+	// ManagedAnnotations holds list of ALB annotations that are managed by rollouts supports multiple annotations
+	ManagedAnnotations = "rollouts.argoproj.io/managed-alb-annotations"
```

Comment:
Is this needed with managedRoutes? Don't they container the same info a list of routes that the rollouts controller manages?

Reply:
No, It's different functionality here.
There is a code for resetting the albAction for non existing rollouts anymore.